sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala

```diff
@@ -46,8 +46,8 @@ class ValidatingPropertyMap[K, V](val m: Map[K, V]) {
 
   def getString(key: K, default: Option[String]): Option[String] = {
```

Comment:
We have getOption now. But I think that the function that control the "" value is better and necessary.